client/src/components/TabBody/style.css

```diff
@@ -0,0 +1,29 @@
+.tabBody-guidelines {
+  text-align: right;
+  font-size: .5em;
+}
+
+.tabBody-guidelines-href {
+  color: #E60085;
+  font-weight: 600;
+}
+
+.tabBody-guidelines-href:hover {
+  color: #08A500;
+}
+
+.tabBody-question {
+  color: #000000;
+  font-size: 1em;
+  font-weight: bold;
+}
+
+.tabBody-answer {
+  color: #000000;
+  font-size: .9em;
+  font-weight: bold;
```

Comment:
same thing here
you don't need all this style it's already in h2 tag

Reply:
I removed them